test/typescript-karma/Dashboards/Component/highcharts.js

```diff
@@ -265,13 +265,8 @@ test('HighchartsComponent resizing', function (assert) {
 
     component.resize(undefined, 300);
 
-    assert.deepEqual(
-        {
-            height: component.element.offsetHeight
-        },
-        {
-            height: 300
-        },
+    assert.ok(
```

Comment:
whenever possible use `assert.stictEqual`- if the error occurs, it's easier to track what the error is and what the values we want

Reply:
In case of checking true/false, the `.ok` is enough, so I'm not convinced in this specific case.